doc/source/examples/multiphysics/dam-break-VOF/dam-break-VOF.rst

```diff
@@ -0,0 +1,255 @@
+==========================
+Dam break VOF
+==========================
+
```

Comment:
Maybe mention the features of the example (similar to 2D Lid-Driven Cavity Flow example). You can mention adaptive time-stepping method as a feature, for instance.

Reply:
+1 Please do so.